app/controllers/goals_controller.rb

```diff
@@ -0,0 +1,98 @@
+class GoalsController < ApplicationController
+
+  def index
+    @messages = Message.broadcast
```

Comment:
It would be railsy to move this @messages = Message.broadcast to a before_action - you do it in most of your routes.

Also, be careful about name clashes - you use @messages to mean something else in the show route - it might be worth making this one @sidebar_messages or similar, so that it's obvious what it's for.